packages/ocolor/ocolor.1.2/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "Print with style in your terminal using Format's semantic tags"
+maintainer: "Marc Chevalier <ocolor@marc-chevalier.com>"
+authors: "Marc Chevalier <ocolor@marc-chevalier.com>"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {build & >= "1.6.3"}
+  "cppo" {build & >= "1.6.5"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
```
You don't seem to use dune substitutions

Reply:
No, indeed. A stupid copy-paste of some documentation. Thanks!